src/errors.ts

```diff
@@ -76,3 +76,17 @@ export class MissingNewJobMessageError extends Error {
     Object.setPrototypeOf(this, MissingNewJobMessageError.prototype)
   }
 }
+
+export class InvalidHash extends Error {
```

Comment:
@yeraydavidrodriguez all errors have the suffix `Error`

Reply:
You are right. Fixed.